snow/networking/benchlist/benchlist.go

```diff
@@ -225,14 +253,15 @@ func (b *queryBenchlist) cleanup() {
 	maxBenchlistWeight := uint64(float64(totalWeight) * b.maxPortion)
 
 	// Iterate over elements of the benchlist in order of expiration
-	for e := b.benchlistOrder.Front(); e != nil; e = e.Next() {
+	for e := b.benchlistOrder.Front(); e != nil; {
```

Comment:
I think this whould just be cleaner being:
```
for b.benchlistOrder.Len() > 0 {
    e := b.benchlistOrder.Front()
```
And then you don't need to do the variable management at the end of the loop.